packages/prosemirror-lwdita-backend/src/api/modules/octokit.module.mts

```diff
@@ -224,8 +225,8 @@ const commitChanges = async (octokit: Octokit, owner: string, repo: string, bran
       tree: treeData.sha,
       parents: [lastCommit.sha],
       committer: {
-        name: "Petal GitHub App",
-        email: "petal@evolvedbianry.com"
+        name: PETAL_COMMITTER_NAME,
```

Comment:
It would be more appropriate if we name it `PetalBot_commiter_name`, we are using PetalBot everywhere now.